### DISCUSSION TITLE
Instrumenta v1.0

### DISCUSSION THREAD

**Discussion by iappyx** [iappyx/Instrumenta#11] [General]
🎉 First major release of Instrumenta!

<hr /><em>This discussion was created from the release <a href='https://github.com/iappyx/Instrumenta/releases/tag/1.0'>Instrumenta v1.0</a>.</em>

**Comment by SjoerdV** [comment]
May I be the first to congratulate you. Awesome work! 
Harvey Balls are back in action and may the Schwartz be with you!
